config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/MagnaliumCrossbo-AAAAAAAAAAAAAAAAAAAEgg==.json

```diff
@@ -67,7 +67,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Magnalium bolts are much better than your old wooden bolts.\n\nYou can use nether quartz on the bolts to make them even better. This requires a free modifier slot.\n\nExperiment with NEI to find other good bolt materials. \n\nTo make it, you must use a Fluid Solidifer instead of the Tinker\u0027s casting table.\n\n§4You\u0027ll need to get the molten iron out of the smeltery and into the fluid solidifier, the GT molten iron won\u0027t work."
+      "desc:8": "Magnalium bolts are much better than your old wooden bolts.\n\nYou can use nether quartz on the bolts to make them even better. This requires a free modifier slot.\n\nExperiment with NEI to find other good bolt materials. \n\nTo make it, you must use a Fluid Solidifer instead of the Tinker\u0027s casting table.\n\nIt\u0027s important that you get the molten iron out of the smeltery and into the fluid solidifier, the GT molten iron won\u0027t work.\n\nPlease keep it in mind that bolts need to be in your inventory in order to be pickable, not in Battlegear slots."
```

Comment:
I dont even know what you are trying to say here. "pickable"?

Reply:
Here's the definition of [pickable](https://www.merriam-webster.com/dictionary/pickable).

My intention in using "pickable" was to say "able to be picked up".

Reply:
Checked on GTNH 2.3.0 with magnalium crossbow and bolts - the bolts are not picked up when the quiver is in any battlegear slot, but are instantly picked up if the quiver is in the inventory.

Reply:
Reworded.